test/data/housing.txt

```diff
@@ -0,0 +1,506 @@
+24.00	0.00632	18.00	2.310	0	0.5380	6.5750	65.20	4.0900	1	296.0	15.30	396.90	4.98
```

Comment:
Would it be reasonable to add a header to the file to help users understand it:
`CrimesPerCapita	PercentResidental	PercentNonRetail	CharlesRiver	NitricOxides	RoomsPerDwelling	PercentPre40s	EmploymentDistance	HighwayDistance	TaxRate	TeacherRatio	BlackIndex	PercentLowIncome	MedianHomeValue`

The headed is also used when producing the feature importance as it will then print the column headers (textual form) instead of just a semi-opaque 'F13'.

Reply:
Done.